### PR TITLE
[MHDP-6] changed redirection method of coursegroup graph

### PR DESCRIPTION
**Pull request by prnvshrn**
When clicking on a course  in the coursegroup graph, a 404 error was showing up. This is because window.location.href , written in the onclick method of the graph, was removing <b>'mbl-data'</b> from <b>'https://amphora.asu.edu/mbl-data/'</b>.

So, I have changed the window.location.href to use Django url in order to give the correct results.

### CODE REVIEW

**Comment by jdamerow** [comment]
we discussed this in the meeting :)

**Comment by prnvshrn** [comment]
I have now changed the redirection method by using split. Split will have the part of the url which is before **'/coursegraph'** (In our case, https://amphora.asu.edu/mbl-data/ ). Then, the clicked course id will be appended to redirect to the desired course without throwing the 404 error.

File: browser/templates/browser/coursegroup.html [hunk 1]
```diff
@@ -168,7 +168,8 @@ <h3>Attendance</h3>
     .remove();
 
   u.on('click', function(d) {
-      window.location.href = '/course/' + d.id + '/';
+      // Split the url to preserve the part before coursegroup and added the clicked course from the graph to url
+      window.location.href = window.location.href.split('/coursegroup')[0] + '/course/' + d.id + '/';
```
Comment (jdamerow) [inline browser/templates/browser/coursegroup.html]: is there a reason you're not using the "courses" url here (with the url tag)?

Comment (prnvshrn) [reply browser/templates/browser/coursegroup.html]: Added courses url, and appended clicked course number from the graph with it

**Pull request merged**